Patch104pZH/GameFilesEdited/Data/INI/ObjectCreationList.ini

```
@@ -6731,7 +6731,7 @@ End
 ObjectCreationList OCL_ChinaTankECMDebris
   CreateObject
     ObjectNames = DeadChinaECMTankHulk
-    Offset      = X:-6.62 Y:0.0 Z:0.0
+    Offset      = X:0.0 Y:0.0 Z:0.0 ; Patch104p @bugfix from X:-6.62 Y:0.0 Z:0.0
```

```suggestion
    Offset      = X:0.0 Y:0.0 Z:0.0 ; Patch104p @bugfix Fix offset.
```
No need to keep this value. It is from copy paste and does not apply to these models.

---

Fixed